help/start-a-call.md

```diff
@@ -56,18 +56,34 @@ supported by zulip are:
 
 !!! tip ""
 
-    It is also possible to disable the video call button for your organization by
-    setting the provider to "None".
+    It is also possible to disable the video call button for your organization
+    by setting the provider to "None".
 
 ### Change your organization's video call provider
 
 {start_tabs}
 
 {settings_tab|organization-settings}
 
-1. Under **Other settings** select appropriate provider from **Call provider** dropdown.
+1. Under **Other settings**, select the desired provider from the
+   **Call provider** dropdown.
 
-1. Click **Save changes**.
+{!save-changes.md!}
+
+{end_tabs}
+
+### Configure a self-hosted instance of Jitsi Meet
+
```

Comment:
This heading doesn't seem quite right. Maybe "Use a self-hosted..."?

Reply:
And then this intro sentence from the integration doc seems helpful here as well:

> Zulip uses the cloud version of [Jitsi Meet](https://meet.jit.si/)
as its default video call provider. You can also use a
self-hosted instance of Jitsi Meet